server/sonar-server/src/main/java/org/sonar/server/util/ClassLoaderUtils.java

```diff
@@ -86,7 +81,7 @@ public boolean apply(@Nullable String path) {
    * @param predicate
    * @return a list of relative paths, for example {"org/sonar/sqale", "org/sonar/sqale/foo", "org/sonar/sqale/foo/bar.txt}. Never null.
    */
-  static Collection<String> listResources(ClassLoader classLoader, String rootPath, Predicate<String> predicate) {
+  public static Collection<String> listResources(ClassLoader classLoader, String rootPath, Predicate<String> predicate) {
     String jarPath = null;
     JarFile jar = null;
     try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this "try" to a try-with-resources. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2093)